src/pages-and-resources/xpert-unit-summary/data/thunks.js

```diff
@@ -10,9 +10,9 @@ export function updateXpertSettings(courseId, state) {
     dispatch(updateSavingStatus({ status: RequestStatus.IN_PROGRESS }));
     try {
       const { response } = await postXpertSettings(courseId, state);
-      const { success, enabled } = response;
+      const { success } = response;
```

Comment:
Not a deal breaker, but a small nit: could you express this more succinctly by using object destructuring?

```
const { response: { success } = {} } = await postXpertSettings(courseId, state);
```

Perhaps I missed it, but it doesn't seem like `response` gets used elsewhere in this block.